src/documents/WebGamesAndSimulations.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Web Games and Simulations
+layout: default
+category: Methods
+---
+
+Interactive games can provide rich learning experiences for many students - but for some, these games are a accessibility and inclusivity challenge. This guide aims at giving some practical techniques to help improve the inclusiveness and accessibility of interactive content whether retrofitting existing content or creating a new game from scratch.
+
+**Scope of this Guide:** This guide deals with interactive games built using standard web technologies such as HTML5, CSS, SVG, and JavaScript. By using established web standards, the content you create has a higher chance of being understood by assistive technologies.
+
+## Orienting the User
+
+Regardless of the modality used for interaction, a logical, predictable structure helps users focus on learning and the experience instead of getting stuck on the interaction. Consistency and predicability is important.
+
+Help orient the user by:
+
+* having a consistent structure.
+* describe the scene and setting so non-visual users can understand their context.
+* identifying the actors and interactive parts.
+* giving meaningful and predictable landmarks for the user.
+* use progress and status indicators as necessary.
+* try to keep the structure stable. Be mindful about removing items completely from the structure - consider disabling elements rather than removing them.
+
```

Comment:
End each bullet point with a semi-colon. Reserve the period for the last bullet point.